projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -4333,6 +4333,12 @@ public string DateOfLastLiveBirth
             }
             set
             {
+                // According to the IJE spreadsheet, if one element in the DateOfLastOtherPregnancyOutcomeYear date is set to 88, then this means there was explicitly "no care" provided. This means the other DateOfLastOtherPregnancyOutcomeYear date elements should also be considered as "no care" since they combine to make one field. In this case, there should be no date data at all since it explicitly never happened, it should not exist in FHIR and should return null. However, the IG doesn't yet have a complete explicit protocol for this case.
+                if (value == 88)
```

Comment:
The all 8s concept comes from IJE, so it should be handled on the IJE side of the library same way we handle all 9s. We don't want someone using the library for a FHIR native system to have to continue using all 8s if its a relic of IJE. 
I'd recommend adding a new helper function in VitalRecord.cs that is like NumericAllowingUnknown_Set and NumericAllowingUnknown_Get but converts all 8s to null or blank. Then update the IJE fields that have all 8s to use that new helper instead of NumericAllowingUnknown_Set/Get.